MODULE.bazel

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 The Bazel Authors.
```

Comment:
Here and in other new files use the year this file is created (2024) instead of 2018